database.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+var knex = require('knex')({
+  client: 'mssql',
+  connection: {
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_DATABASE
+  }
```

Comment:
This is perfect and what I was talking about

Reply:
Salam Ahmed, thanks for reminding me of installing mssql. It is also mentioned in Knex documents, but I did not know how the app was working as I have tried it twice and it created me the database and the table.
Anyway, good point the dependency is being added now. I will push my change soon.